src/components/squareComponent/square.spec.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import GameSquare from './squareComponent';
+import GameBoard from '../boardComponent/boardComponent';
+import Enzyme from 'enzyme';
+import {shallow} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Square Component', () => {
+
+    it('renders the Square component without crashing', () => {
+        const gameSpaceDiv = document.createElement('div');
+        ReactDOM.render(<GameSquare />, gameSpaceDiv);
+        ReactDOM.unmountComponentAtNode(gameSpaceDiv);
+      });
+
+      it('renders one Square component within the Board componenet without crashing', () => {
+        const wrapper = shallow(<GameBoard />);
+        const boardGridDiv = wrapper.find(GameSquare);
+        expect(wrapper.contains(<GameSquare />)).toBe(true);
+      });
+
+      it('tests that each space on the empty board is clickable', () => {
+        const wrapper = shallow(<GameSquare />);
+        const boardDiv = wrapper.find(GameSquare);
+        const clickSquare = wrapper.find('a').simulate('click');
+        expect(clickSquare.length).toEqual(1);
```

Comment:
how useful is this test? Does anything actually happen when that 'a' is clicked?

Reply:
I suppose not too useful. Especially if I'm removing the a tag.

Reply:
👍 to @akiraandy 

It's usually best to hold off implementing features you don't need yet. In this case, that includes making squares clickable.

Reply:
Okay, got it, I can remove this feature and hold off on it for now